src/api/public/apidocs-new/paths/staging_project_name_staging_projects_staging_project_name_staged_requests.yaml

```diff
@@ -0,0 +1,199 @@
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            With Excluded Requsets:
+              value:
+                code: invalid_request
+                summary: |
+                  Request ID currently excluded from project project_name. Use --remove-exclusion if you want to force this action.
+            Unknown Request:
+              value:
+                code: invalid_request
+                summary: "Request ID doesn't exist"
+            Invalid XML Format:
+              value:
+                code: invalid_xml_format
+                summary: 'XML format is not valid'
+            Empty Body:
+              value:
+                code: invalid_request
+                summary: 'Empty body'
+    '404':
+      description: Not Found
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            Without Staging Workflow:
+              value:
+                code: not_found
+                sumary: "Project project_name doesn't have an associated Staging Workflow"
+            Unknown Staging Project:
+              value:
+                code: not_found
+                summary: "Staging Project 'project_name:Staging:G' not found in Staging: 'project_name'"
+            Unknown Project:
+              value:
+                code: not_found
+                summary: "Project 'project_name' not found."
+    '424':
+      description: Unacceptable State
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            Unacceptable State:
+              value:
+                code: staging_project_not_in_acceptable_state
+                sumary: "Can't change staged requests: Project 'project_name' is being accepted."
+  tags:
+    - Staging Workflow
+
+delete:
+  summary: Delete requests from the staging project.
+  description: Delete the given requests from the staging project they were staged in.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/staging_project_name.yaml'
+  requestBody:
+    description: List of requests to be removed from the staging project.
+    content:
+      application/xml; charset=utf-8:
+        schema:
+          $ref: '../components/schemas/staging/requests_to_stage.yaml'
+        examples:
+          Multiple Requests:
+            value:
+              request:
+                - id: 1
+                - id: 2
+  responses:
+    '400':
+      description: |
+        Bad Request.
+
+        XML Schema used for body validation: [status.xsd](../schema/status.xsd).
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
```

Comment:
Starting with the data that obs's `rake dev:test_data:create` creates, request #1 is not staged, if I try to remove that request, I get:
```
<status code="invalid_request">
  <summary>Error while unassigning requests: Request '1' is not staged and Requests with number: 2 don't belong to Staging: home:Admin</summary>
</status>
```

Reply:
They are two concatenated errors, I described them separately